### ISSUE TITLE
Discarding an activity does not discard its mirrored bookmark

### ISSUE THREAD

**Issue by shipmints** [alphapapa/activities.el#96]
Seems easy enough to reproduce. Create an activity, wait for the time interval for bookmarks to be saved and also activities to persist. Discard an activity. The bookmark remains behind.

Shite, sorry. While I am writing this, I took a look at the source code for ```activities-discard```, it says:

```elisp
  ;; TODO: Discard relevant bookmarks when `activities-bookmark-store' is enabled.
```

I guess this issue can stand as a reminder to contributors to write that code. It could be me if I can learn the necessary elisp skills.

**Comment by alphapapa** [comment]
Yeah, just haven't bothered to do it yet.  Wasn't sure if the automatically created bookmarks would be universally appreciated, anyway.  Patches welcome, or I'll get to it eventually.  If you do want to try your hand at it, I'll be glad to assist with Elispisms.

**Comment by shipmints** [comment]
I hear you. If activities bookmarks aren't long for this world, it wouldn't kill me. I prefer using the activities API itself vs. bookmarks despite them being a natural part of the current implementation.

**Comment by alphapapa** [comment]
I don't think I'll be removing the feature, but I might be convinced to disable it by default.

**Comment by shipmints** [comment]
Giving this a shot.

**Comment by shipmints** [comment]
See #101.

**Comment by shipmints** [comment]
Finally getting back to this. Pushed updates accommodating your comments.
